src/pytest_cov/plugin.py

```diff
@@ -341,7 +341,7 @@ def pytest_terminal_summary(self, terminalreporter):
             markup = {'red': True, 'bold': True} if failed else {'green': True}
             message = (
                 '{fail}Required test coverage of {required}% {reached}. '
-                'Total coverage: {actual:.2f}%\n'
+                'Total coverage: {actual}%\n'
```

Comment:
I think this would fix the problem you described without changing the output too dramatically.
```suggestion
                'Total coverage: {truncate(actual, 2)}%\n'
```

Reply:
Would be cleaner imo to rely on what coverage is doing: https://github.com/nedbat/coveragepy/blob/a1d8d2961c9793cf8d3fa09666012190cdec9214/coverage/results.py#L253

Reply:
Right !

Reply:
Maybe we could just reuse that from coverage? Hopefully @nedbat won't change it soon :-)

Reply:
I am actively refactoring that file to make more of it public: https://github.com/nedbat/coveragepy/tree/nedbat/public-analysis . We could talk about how to make it useful here.